package.json

```diff
@@ -50,6 +50,7 @@
     "eslint-plugin-jsx-a11y": "^5.1.1",
     "eslint-plugin-react": "^7.4.0",
     "istanbul": "^0.4.5",
+    "mdi": "^2.0.46",
```

Comment:
Se não for usar a biblioteca de icones, pode desinstalar do projeto, menos dependencias pro yarn install